### PR TITLE
Highlight lighthouse basalt pillar agility obstacles

### LINKED ISSUE #11721: Agility obstacles of Rellekka Lighthouse
**Is your feature request related to a problem? Please describe.**
The stepping stones from the barbarian outpost -> lighthouse do not have the green agility boxes.

**Describe the solution you'd like**
The correct path on the stepping stones should be pathed with green boxes

image

Using devtools in runelite, I found the right stones and tried to add them in the same style, but I'm new to Java and based on some errors, I'm probably doing something I shouldn't. I believe that these lines should be added to the [Obstacles file](https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java) below the additions for Prif.

```
		// Rellekka lighthouse
		BEACH_4550, BASALT_ROCK_4551, BASALT_ROCK_4552, BASALT_ROCK_4553, BASALT_ROCK_4554,
		BASALT_ROCK_4555, BASALT_ROCK_4556, BASALT_ROCK_4557, BASALT_ROCK_4558, ROCKY_SHORE_4559
```

### PR DESCRIPTION
**Pull request by ComfortCurry**
Closes #11721

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java [hunk 1]
```diff
@@ -94,7 +94,10 @@
 		ZIP_LINE_11645, ZIP_LINE_11646,
 		// Prifddinas
 		LADDER_36221, TIGHTROPE_36225, CHIMNEY_36227, ROOF_EDGE, DARK_HOLE_36229, LADDER_36231, LADDER_36232,
-		ROPE_BRIDGE_36233, TIGHTROPE_36234, ROPE_BRIDGE_36235, TIGHTROPE_36236, TIGHTROPE_36237, DARK_HOLE_36238
+		ROPE_BRIDGE_36233, TIGHTROPE_36234, ROPE_BRIDGE_36235, TIGHTROPE_36236, TIGHTROPE_36237, DARK_HOLE_36238 // ,
+		// Rellekka lighthouse
+		// BEACH_4550, BASALT_ROCK_4551, BASALT_ROCK_4552, BASALT_ROCK_4553, BASALT_ROCK_4554,
+		// BASALT_ROCK_4555, BASALT_ROCK_4556, BASALT_ROCK_4557, BASALT_ROCK_4558, ROCKY_SHORE_4559
```
Comment (Broooklyn) [inline runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java:100]:
```suggestion
		// Rellekka Lighthouse
		BASALT_ROCK, BASALT_ROCK_4553, BASALT_ROCK_4554, BASALT_ROCK_4556, BASALT_ROCK_4558
```

A lot of the objects you highlighted were superfluous, in that clicking one can bring you both ways. You can simplify it and achieve the same result with less clutter.

Screen Shot 2020-05-28 at 2 20 46 AM

Comment (ComfortCurry) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java:100]: Cool, when I was going through I found that to jump to the next tile, you could click either your current tile OR the next one. I like your decluttered way as opposed to the "all the options" way

Comment (ComfortCurry) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java:100]: I just noticed that to get the code working, you had to put BASALT_ROCK before any BASALT_ROCK_nxyz. Is this a convention specific to runelite, or is it related to how variables are initialized in java?

Comment (mcnabbian) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: @ComfortCurry I'm a newb to java and also curious about this. I did find 
`public static final int BASALT_ROCK = 4551;
public static final int BASALT_ROCK_4552 = 4552;
public static final int BASALT_ROCK_4553 = 4553;` 
in the runelite api. Maybe that has something to do with it?

Comment (ComfortCurry) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/Obstacles.java]: > 
> 
> `public static final int BASALT_ROCK = 4551; public static final int BASALT_ROCK_4552 = 4552; public static final int BASALT_ROCK_4553 = 4553;`
> in the runelite api. Maybe that has something to do with it?

I got onto the #Development channel in discord and they were super helpful, and explained this to me. I did my naming based on the trend I saw in Obstacles.java, but turns out the naming schema I inferred has an exception: the first time an item appears in the [itemID list](https://github.com/runelite/runelite/tree/master/runelite-api/src/main/java/net/runelite/api), it doesn't have the item# appended. BASALT_ROCK_4551 doesn't exist. The disc folks gave me a tip for problems like this in the future: shift-clicking an object shows where it comes from. Hope this helps!

**Comment by Nightfirecat** [comment]
While you can hop each gap by clicking on the objects on either side, we may as well highlight the ones on either side.

**Comment by Adam-** [comment]
added in 91bac5821a1adb172157bd45e2f0d6eae7185b04

**Pull request closed without merging**